### PR TITLE
1.4.1 release prep

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

**Comment by hunner** [comment]
This appears to be prep plus fastforwarding 1.4.x to master.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+##Supported Release 1.4.1
```
Comment (hunner) [inline CHANGELOG.md]: Hash-section syntax is supposed to have a space between the hashes and the text.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+###Summary
+This release contains bugfixes around certificate chains and other testing improvements.
+
+####Bugfixes
+- Dont expose keystore content when keystore initally empty.
+- Support certificate chains in certificate file.
+- Support multiple intermediate certificates in chain.
+
+####Improvements
```
Comment (hunner) [inline CHANGELOG.md]: These would be considered bugfixes, since they are not "Features" or "Changes"

File: CHANGELOG.md (hunk 1 above)
Comment (tphoney) [inline CHANGELOG.md]: this is done through out the changelog, should i fix it all or follow what is there?

File: CHANGELOG.md (hunk 2 above)
Comment (tphoney) [inline CHANGELOG.md]: are you talking about the improvements ?

File: CHANGELOG.md (hunk 1 above)
Comment (hunner) [inline CHANGELOG.md]: http://spec.commonmark.org/0.24/#atx-headings

**Comment by tphoney** [comment]
cheers @hunner

**Comment by johngmyers** [comment]
MODULES-3023 is arguably a feature

**Pull request merged**